packages/cli/src/baseCommand.js

```diff
@@ -0,0 +1,58 @@
+const { Command } = require('@oclif/command');
+const { isEmpty } = require('lodash');
+const { cli: ux } = require('cli-ux');
+
+const { readCredentials, checkCredentials } = require('./utils/api');
+
+class ZapierBaseCommand extends Command {
+  _version() {
+    return 'blah blah ok'; // this should overwrite the version string, but doesn't?
+  }
+
+  log(message) {
```

Comment:
instead of doing `context.line`, we can run `this.log` from inside the commands. This means that all of our log lines will need to be in the commands themselves, but that'll probably help us split them apart and make them smaller.